vcd/resource_vcd_network_routed_v2.go

```diff
@@ -211,10 +227,20 @@ func resourceVcdNetworkRoutedV2Read(ctx context.Context, d *schema.ResourceData,
 
 	d.SetId(orgNetwork.OpenApiOrgVdcNetwork.ID)
 
+	metadata, err := orgNetwork.GetMetadata()
+	if err != nil {
+		log.Printf("[DEBUG] Unable to find routed network v2 metadata: %s", err)
+		return diag.Errorf("[routed network read v2] unable to find Org VDC network metadata %s", err)
```

Comment:
```suggestion
		return diag.Errorf("[routed network read v2] unable to find Org VDC routed network metadata %s", err)
```
Let's be more specific in all places

Reply:
Fixed!